west.yml

```diff
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: aescolar
```

Comment:
We'll have a zephyrproject-rtos fork of this, right?

Reply:
That's the plan